### ISSUE TITLE
Outlier steps

### ISSUE THREAD

**Issue by kaitlyngaynor** [kaitlyngaynor/hopland-hunting#12]
@amcinturff I calculated step lengths for all of the 304 tracks (after they were cleaned to a 3 minute fix interval). I'm finding several outliers. The largest step lengths are unrealistic—133, 130, 79, 58, 50, 35, 30, 23, 12 kilometers... then there is a gap in the distribution and we see 5, 4, etc. Do you know if any data cleaning has been applied to these data? I'm assuming not. I think we should throw out "bad points" as is typical for animal movement analyses.

Here are the problem individuals:
- 81416_12 (6 bad points) - I think we should just ditch this one entirely; there are only 20 points in total, and 6 of them are out in the middle of nowhere
- 81416_10 (2 bad points)
- 080815_08 (1 bad point)
- 080815_09 (1 bad point)

Also there were a few hunters who just left the property! They account for nearly all of the ~5km step lengths (going 60mph would result in a step length of 5km in 3 minutes). I guess we should break their tracks in two.

- 081316_11
- 081316_12

<img width="911" alt="Screen Shot 2021-12-06 at 10 04 41 PM" src="https://user-images.githubusercontent.com/29489999/144975416-e438f3e9-e346-4cf0-b3b4-7205619b45ba.png">

**Comment by kaitlyngaynor** [comment]
For 081316_11, one of two individuals who left the study area, I have duplicated the tracks and renamed _1 and _2, and then updated (manually) the igotu_metadata_cleaned so there are two rows for each track, one _1 and one _2, with the start/end times for the morning hunt and the evening hunt. I determined these start/end times manually by looking at the tracks and seeing when they left HQ mid-day, and when they returned. The cleaning script should take care of the rest. I excluded 081316_12 from the study (see issue #13)

Still need to deal with the other extreme outliers.
